$MirOS: ports/emulators/zsnes/patches/patch-src_configure_in,v 1.2 2005/10/02 19:25:35 tg Exp $
--- src/configure.in.orig	Wed Jan 19 22:52:16 2005
+++ src/configure.in	Thu Oct 30 19:38:13 2008
@@ -9,13 +9,12 @@ AC_SUBST(VERSION)
 AC_ARG_WITH(nasm-prefix,[  --with-nasm-prefix=PFX Prefix where nasm is installed (optional)], nasm_prefix="$withval", nasm_prefix="")
 
 dnl -- Where is our compiler, and who are we compiling for?
-CFLAGS="$CFLAGS -pipe -I. -Wall -I/usr/local/include -I/usr/include"
+CFLAGS="$CFLAGS $CPPFLAGS -I. -Wall"
 
 dnl -- You might wanna change -I/usr/include if you're trying to do this with
 dnl Mingw, because it doesnt have a standard file tree. Maybe if ming do this
 dnl else do -I/usr/include. Or some shit. You figure it out.
 
-LDFLAGS="$LDFLAGS -L/usr/local/lib -L/usr/lib"
 
 dnl -- Same thing here.
 
@@ -37,9 +36,9 @@ case "$target" in
 		CFLAGS="$CFLAGS -D__LINUX__ -D__BEOS__"
         	NFLAGS="$NFLAGS -D__LINUX__ -D__BEOS__ -f elf -DELF"
 		;;
-	*-*-*openbsd*)
+	*-*-*openbsd* | *-*-mirbsd*)
 		CFLAGS="$CFLAGS -D__LINUX__ -D__FreeBSD__"
-		NFLAGS="$NFLAGS -D__LINUX__ -D__FreeBSD__ -D__OpenBSD__ -f aoutb"
+		NFLAGS="$NFLAGS -D__LINUX__ -D__FreeBSD__ -D__OpenBSD__ -DELF -f elf"
 		;;
 	*-*-*bsd*)
 		CFLAGS="$CFLAGS -D__LINUX__ -D__FreeBSD__"
@@ -113,13 +112,15 @@ if test x$debug = xyes; then
 	dnl enabled. 
 
 	CFLAGS="$CFLAGS -DDEBUG -O0 -fno-omit-frame-pointer -ggdb3"
-	NFLAGS="$NFLAGS -DDEBUG -g -s -O1" dnl -O0 doesnt work
+	NFLAGS="$NFLAGS -DDEBUG -s -O1" dnl -O0 doesnt work
+	NDFLAGS="-g"
 	ZSNESEXE="zsnesd"
 else
 	AC_MSG_RESULT(no)
 	
 	CFLAGS="$CFLAGS -O3 -ffast-math -fomit-frame-pointer -fexpensive-optimizations -s"
 	NFLAGS="$NFLAGS -O1"
+	NDFLAGS=
 	ZSNESEXE="zsnes"
 fi
 
@@ -156,6 +157,7 @@ dnl Checks for library functions.
 AC_SUBST(ZSNESEXE)
 AC_SUBST(GL_DRAW)
 AC_SUBST(NFLAGS)
+AC_SUBST(NDFLAGS)
 AC_SUBST(NASMPATH)
 AC_OUTPUT(Makefile)
 echo
